### ISSUE TITLE
Basic question about Protein.Group in pg_matrix

### ISSUE THREAD

**Issue by cxdummies** [vdemichev/DiaNN#1149]
Dear Vadim,

I would like to ask a very basic question about Protein.Group in dia-NN. Below is an example of the protein groups extracted from the pg_matrix.

Protein.Group                             Protein.Names	                                                           Genes	        First.Protein.Description
Q9Y6Y8	                                  S23IP_HUMAN	                                                           SEC23IP	SEC23-interacting protein
Q9Y6Y8;A0A994J542	          A0A994J542_HUMAN;S23IP_HUMAN	                           SEC23IP	Isoform of Q9Y6Y8, 
Q9Y6Y8;A0A994J542;H7C0V8	  A0A994J542_HUMAN;H7C0V8_HUMAN;S23IP_HUMAN   SEC23IP	Isoform of Q9Y6Y8,

My understanding of a protein group was that the first protein of the group would account for all the detected peptides that had been assigned to that group. However, in the example above, Q9Y6Y8 is the first protein for all three groups.

1) Does this mean that in dia-NN it is rather the last protein of each protein group that carries the unique peptide? I observed the same Protein.Group structure in the output from the diann_maxlfq() function.

2) Are the corresponding protein quantities then specific for the last protein of each protein group? In the above example, are the protein quantities specific for Q9Y6Y8, then for A0A994J542 and the last one for H7C0V8?

Looking forward to your feedback. Thank you.

**Comment by vdemichev** [comment]
Hi,

This looks like output obtain with 'Heuristic protein inference' disabled. The meaning of protein groups in this case is the following: each protein group has quantities associated with it which estimate the joint contribution of proteins listed in the group, in unknown proportions, quantification of the group is performed using precursors matched to the group in the main report.

1. No, order of proteins in the group does not have a particular meaning.
2. No.

Best,
Vadim

**Comment by vdemichev** [comment]
Yes, for typical analysis better to keep on, just keep in mind the limitations of protein inference via maximum parsimony.

1. No.
2. Not sure what MaxQuant does. However if you need isoform-specific quantification, the steps would be: (i) set protein inference to 'isoforms' in the GUI, (ii) filter the main output for proteotypic peptides and (iii) requantify proteins using the iq R package and only those peptides.

Best,
Vadim
